conda/testing/fixtures.py

```diff
@@ -161,3 +186,287 @@ def _solver_helper(
     assert context.solver == solver
 
     yield solver
+
+
+@pytest.fixture(scope="session")
+def global_capsys(request) -> Iterator[MultiCapture]:
```

Comment:
```suggestion
def session_capsys(request) -> Iterator[MultiCapture]:
```

"global" having a particular meaning in Python might not be what we want in this case